src/main/java/teammates/common/util/Templates.java

```diff
@@ -233,9 +233,12 @@ public static String populateTemplate(String template, String... keyValuePairs)
             public static final String GENERATED_OPTIONS = "${Const.ParamsNames.FEEDBACK_QUESTION_GENERATEDOPTIONS}";
             public static final String GENERATE_OPTIONS_FOR_VALUE = "${generateOptionsForValue}";
             public static final String STUDENT_SELECTED = "${studentSelected}";
+            public static final String STUDENT_EXCLUDING_SELF_SELECTED = "${studentExcludingSelfSelected}";
             public static final String TEAM_SELECTED = "${teamSelected}";
             public static final String INSTRUCTOR_SELECTED = "${instructorSelected}";
             public static final String STUDENTS_TO_STRING = "${FeedbackParticipantType.STUDENTS.toString()}";
+            public static final String STUDENTS_EXCLUDING_SELF_TO_STRING = "${FeedbackParticipantType."
+                    + "STUDENTS_EXCLUDING_SELF.toString()}";
```

Comment:
Rather than the awkward breaking at `.`, let's just move the entire String to the next line?